resources/quality/hms434/hms434_global_Super_Coarse_Quality.inst.cfg

```diff
@@ -1,16 +0,0 @@
-[general]
```

Comment:
If you remove or rename profiles, you also need to add version upgrade to make sure that the machines using those removed/renamed profiles in the old version can be upgraded correctly to use the correct profiles in the new version. I will merge this and then make the version upgrade.

Reply:
Version upgrade made in 1977a865f1b0d7ebf301603a4c2e1323e1e5b695